client/blocks/login/error-notice.jsx

```diff
@@ -53,6 +57,12 @@ class ErrorNotice extends Component {
 		);
 	}
 
+	registerSignupUrlClickEvent = () => {
+		this.props.recordTracksEvent( 'calypso_login_sign_up_link_click', {
+			reason: 'unknown_user',
```

Comment:
Is this event prop valuable?

Reply:
if this `unknow_user` error can occur at other places where `calypso_login_sign_up_link_click` track is recorded then it might be more valuable to give additional context to the reason for clarity. If not it should be good as is.